### PR TITLE
Swedish MT STS-B

### PR DESCRIPTION
**Pull request by timpal0l**
Added a Swedish machine translated version of the well known STS-B Corpus

### CODE REVIEW

File: datasets/stsb_mt_sv/stsb_mt_sv.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  author={Isbister, Tim and Sahlgren, Magnus},
+  journal={arXiv preprint arXiv:2009.03116},
+  year={2020}
+}
+"""
+
+_DESCRIPTION = "Machine translated Swedish version of the original STS-B (http://ixa2.si.ehu.eus/stswiki)"
+
+
+class SwedishTranslatedSTSB(datasets.GeneratorBasedBuilder):
```
Comment (joeddav) [inline datasets/stsb_mt_sv/stsb_mt_sv.py:43]:
```suggestion
class StsbMtSv(datasets.GeneratorBasedBuilder):
```
The class name has to match the filename but in camelcase form

File: datasets/stsb_mt_sv/README.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+languages:
+- sv
+licenses:
+- unknown
+multilinguality:
+- monolingual
+size_categories:
+- n<1K
+source_datasets:
+- original
```
Comment (joeddav) [inline datasets/stsb_mt_sv/README.md:15]:
```suggestion
- extended|other-sts-b
```

File: datasets/stsb_mt_sv/README.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
+---
+annotations_creators:
+- crowdsourced
+language_creators:
+- crowdsourced
```
Comment (joeddav) [inline datasets/stsb_mt_sv/README.md:5]:
```suggestion
- crowdsourced
- machine-generated
```

**Review by joeddav — CHANGES_REQUESTED** [review]
Nice work! I added a couple small requested changes that you can accept but beside that I think this looks good to me!

**Review by joeddav — APPROVED** [review]

File: datasets/stsb_mt_sv/dataset_infos.json [hunk 4]
```diff
@@ -0,0 +1 @@
...{2020}\n}\n", "homepage": "https://github.com/timpal0l/sts-benchmark-swedish", "license": "", "features": {"sentence1": {"dtype": "string", "id": null, "_type": "Value"}, "sentence2": {"dtype": "string", "id": null, "_type": "Value"}, "score": {"dtype": "float32", "id": null, "_type": "Value"}}, "post_processed": null, "supervised_keys": null, "builder_name": "swedish_translated_stsb", "config_name": "plain_text", "version": {"version_str": "1.0.0", "description": "", "major": 1, "minor": 0, "patch": 0}, "splits": {"test": {"name": "test", "num_bytes": 171823, "num_examples": 1379, "dataset_name": "swedish_translated_stsb"}, "validation": {"name": "validation", "num_bytes": 218843, "num_examples": 1500, "dataset_name": "swedish_translated_stsb"}, "train": {"name": "train", "num_bytes": 772847, "num_examples": 5749, "dataset_name": "swedish_translated_stsb"}}, "download_checksums": {"https://raw.githubusercontent.com/timpal0l/sts-benchmark-swedish/master/data/stsb-mt-sv.zip": {"num_bytes": 383047, "checksum": "f7f941887a6fea53efa4379c0a5338ee0ea0b704debabbd2dd8dec9c2ef4891f"}}, "download_size": 383047, "post_processing_size": null, "dataset_size": 1163513, "size_in_bytes": 1546560}}
```
Comment (lhoestq) [inline datasets/stsb_mt_sv/dataset_infos.json:1]: this file needs to be regenerated after the class name update

Comment (lhoestq) [reply datasets/stsb_mt_sv/dataset_infos.json:1]: Could you regenerate it and open a PR @ timpal0l please ?
sorry we didn't spot it earlier

Comment (timpal0l) [reply datasets/stsb_mt_sv/dataset_infos.json:1]: @lhoestq Thanks ---> #1138

**Pull request merged**